src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -276,10 +283,11 @@ static ucs_status_t uct_cuda_ipc_iface_query(uct_iface_h tl_iface,
     iface_attr->cap.get.align_mtu       = iface_attr->cap.get.opt_zcopy_align;
     iface_attr->cap.get.max_iov         = 1;
 
-    iface_attr->latency                 = ucs_linear_func_make(1e-6, 0);
+    iface_attr->latency                 = ucs_linear_func_make(
+            iface->config.latency, 0);
     iface_attr->bandwidth.dedicated     = 0;
     iface_attr->bandwidth.shared        = iface->config.bandwidth;
-    iface_attr->overhead                = 7.0e-6;
+    iface_attr->overhead                = iface->config.overhead;
```

Comment:
These changes caused failures in CI wireup checks. @brminich @rakhmets @yosefe is that OK to leave these values unrelated from new config fields and set them back to the old values, WDYT? AFAIU it is impossible to distinguish the peer version at UCT side, right?

Reply:
pls set them back with the comment

Reply:
I think it's ok to leave the values. Because `uct_cuda_ipc_estimate_perf` is used for the protocol selection,

Reply:
Thanks for you opinions, reverted it back with a comment